src/cltk/dependency/spacy_wrapper.py

```diff
@@ -120,6 +123,28 @@ def _download_model(self) -> None:
                     "https://huggingface.co/latincy/la_core_web_lg/resolve/main/la_core_web_lg-any-py3-none-any.whl",
                 ]
             )
+        elif self.language == "grc":
+            # Use OdyCy's Spacy models, hosted on HuggingFace
+            subprocess.check_call(
+                [
+                    "pip",
+                    "install",
+                    "https://huggingface.co/chcaa/grc_odycy_joint_sm/resolve/main/grc_odycy_joint_sm-any-py3-none-any.whl",
+                ]
+            )
+        else:
```

Comment:
🤷‍♂️ shrugs in Python 🤷‍♂️

Reply:
I don't know what shrugs mean. And I can see here duplicate code with Latin. I'm going to fix it.

Reply:
I am just being silly. I think this is the best of all the possible solutions.